Controller/MediaAdminController.php

```diff
@@ -109,6 +117,116 @@ public function listAction(Request $request = null)
     }
 
     /**
+     * @throws \Symfony\Component\Security\Core\Exception\AccessDeniedException
+     *
+     * @return \Symfony\Bundle\FrameworkBundle\Controller\Response|\Symfony\Component\HttpFoundation\Response
+     */
+    public function multiUploadAction(Request $request)
+    {
+        if (false === $this->admin->isGranted('CREATE')) {
+            throw new AccessDeniedHttpException('Access denied');
+        }
+
+        $providerName = $this->getRequest()->get('provider');
+        $provider = $this->get($providerName);
+
+        // NEXT_MAJOR: remove $supportMethodCallable with the next major release.
+        $supportMethodCallable = method_exists($provider, 'supportsMultiUpload');
```

Comment:
couldn' we do this in line 149 directly instead of instantiating a one time used variable?